site/styles/_meta-sub-item.scss

```diff
@@ -37,14 +35,17 @@
   }
 
   .synopsis {
+    animation: fadein 2s;
     font-size: 16px;
     font-weight: $font-weight-light;
     grid-column: 1 / 3;
     letter-spacing: 0.6px;
     line-height: 1.25;
     margin-top: 5px;
+    overflow: hidden;
 
     @include media-breakpoint-up(sm) {
+      display: block !important;
```

Comment:
You can probably drop `!important` if you get rid of [this horrible Bootstrap class](https://github.com/shift72/core-template/blob/b359371cca68a2a1f11f2919898c8272cffb6cb0/site/styles/_shift72.scss#L305)... not sure if that will affect other parts of the site though.  And in terms of avoiding FOUC, would `visibility: hidden/visible` be a more direct solution?... might be too late to go with that now though.

Reply:
I might leave this one as is. Defo have no clarity on any unintended consequences.

And I can't see any FOUC on synopsis? It should be taken care of by the inline `s72-hide`